libxw/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# https://pip.pypa.io/en/stable/user_guide/#requirements-files
```

Comment:
If you're curious about why we need this in the first place, I'd recommend [this blog post](https://caremad.io/posts/2013/07/setup-vs-requirement/) by the current maintainer of [PyPI](https://pypi.org).